src/components/BorrowerProfile/LendersList.vue

```diff
@@ -0,0 +1,138 @@
+<template>
+	<div ref="wrapper"
+		:class="['tw-mt-1.5', 'lg:tw-mb-1.5']"
+	>
+		<div
+			:class="[
+				'tw-w-full',
+				'tw-flex tw-flex-col',
+				{
+					'md:tw-static': !isSticky,
+				},
+				'tw-relative',
+				'lenders-container'
+			]"
+		>
+			<span
+				class="tw-inline-block tw-align-middle tw-mb-2"
+				data-testid="bp-lend-cta-powered-by-text"
+				key="numLendersStat"
+			>
+				<kv-material-icon
+					class="tw-w-2.5 tw-h-2.5 tw-pointer-events-none tw-inline-block tw-align-middle"
+					:icon="mdiLightningBolt"
+				/>
+				{{ poweredByText }}
+			</span>
+			<div v-for="(lender, idx) in sortedLenders" :key="lender.id">
+				<template v-if="isValidLength(idx)">
+					<lenders-list-item
+						:lender="lender"
+					/>
+				</template>
+			</div>
+		</div>
+		<div v-if="!isMobile" class="tw-mt-2">
+			<button
+				class="tw-bg-transparent tw-border tw-border-black tw-px-3 tw-py-1 tw-rounded tw-text-base"
+				@click="toogleLightBox"
+			>
+				See all lenders
+			</button>
+		</div>
+	</div>
+</template>
+
+<script>
+import { mdiLightningBolt } from '@mdi/js';
+import LendersListItem from '@/components/BorrowerProfile/LendersListItem';
+import KvMaterialIcon from '~/@kiva/kv-components/vue/KvMaterialIcon';
+
+export default {
+	name: 'LendersList',
+	inject: ['apollo', 'cookieStore'],
+	props: {
+		lenders: {
+			type: Array,
+			default: () => []
+		},
+		isMobile: {
+			type: Boolean,
+			default: false
+		},
+		numLenders: {
+			type: Number,
+			default: 0
+		}
+	},
+	components: {
+		LendersListItem,
+		KvMaterialIcon,
+	},
+	data() {
+		return {
+			mdiLightningBolt,
+		};
+	},
+	methods: {
+		isValidLength(idx) {
+			return (this.isMobile && idx < 2) || (!this.isMobile && idx < 3);
+		},
+		toogleLightBox() {
+			this.$emit('tooglelightbox');
+		}
+	},
+	computed: {
+		isSticky() {
+			return true;
```

Comment:
Will this ever be more than `return true`? If not, this computed can be removed.

Reply:
Nice catch! I'll remove it